fkbeta/fk/tests.py

```diff
@@ -24,8 +25,8 @@ def test_video_listing(self):
     def test_video_detail(self):
         r = self.client.get('/video/1/')
         self.assertContains(
-            r, 'First broadcast: <i>Jan. 1, 2015, 10 a.m.</i>', count=1)
-        self.assertContains(r, '<h1>Video not available</h1>', count=1)
+            r, _('First broadcast:') + ' <i>', count=1)
```

Comment:
Uhm. Should we run the tests with translation on? And I think you removed the useful bit of the test which is the rendering of the date :)